### PR TITLE
test: Move compliance On Recreate test to On Edit

### PR DESCRIPTION
**Pull request by regexowl**
This migrates a test from previously used On Recreate to current On Edit.

### CODE REVIEW

**Comment by regexowl** [comment]
This moves and enables one of the disabled tests. We'll need to re-enable and update more, but let's take it one step at a time for now.

File: src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx [hunk 1]
```diff
@@ -206,4 +191,30 @@ describe('OpenSCAP edit mode', () => {
...
+    });
+    user.click(packagesNavBtn);
+    await screen.findByRole('heading', {
+      name: /Additional packages/i,
+    });
+    const selectedBtn = await screen.findByRole('button', {
+      name: /Selected/i,
+    });
+    user.click(selectedBtn);
+    await screen.findByText('neovim');
```
Comment (ezr-ondrej) [inline src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx:218]: This changed from `nftables` to `neovim` :thinking:
I know nada about the oscap, so unsure what's happening, thus I figured I'd rather ask ;)

We should change the comment above at least I guess :)

Comment (regexowl) [reply src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx:218]: Yeah, might look a bit chaotic :sweat_smile: The test got disabled during work on Blueprints and in the meantime the mocks changed. This should be the object that gets returned as a mocked response from the commit that made the changes: https://github.com/osbuild/image-builder-frontend/commit/a1bdcaa54ff5e46e63b82c9c99f3a36fd523b4bc#diff-07b400aeb49d2de7ed7da9354500d974cacc41ea79688a0def4ac864b9bed1b0L17-L43

'nftables' were removed from packages together with other and replaced by 'neovim' only.

Great point about the comment, I'll update it right away!

Comment (ezr-ondrej) [reply src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx:218]: Ah, gotcha, thanks for explaining :orange_heart:

**Review by ezr-ondrej — APPROVED** [review]
Thanks! :orange_heart:

**Pull request merged**